_data/collection-index.yml

```
@@ -243,3 +243,8 @@
   contact: https://github.com/ChristopherMacGown/devcontainer-features/issues
   repository: https://github.com/ChristopherMacGown/devcontainer-features
   ociReference: ghcr.io/christophermacgown/devcontainer-features
+  - name: Pwd9000-ML devcontainer templates
+  maintainer: Marcel Lupo @Pwd9000-ML
+  contact: https://github.com/Pwd9000-ML/devcontainer-templates/issues
+  repository: https://github.com/Pwd9000-ML/devcontainer-templates
+  ociReference: ghcr.io/Pwd9000-ML/devcontainer-templates
```

```suggestion
  ociReference: ghcr.io/pwd9000-ml/devcontainer-templates
```

The ORAS spec requires the `ociReference` to be in lower case. Can we update here? 

The supporting tools does help convert the cases, however, its better to sync as much as possible.